packages/core/echo/echo-schema/src/database.test.ts

```diff
@@ -72,33 +72,6 @@ describe('EchoDatabase', () => {
     expect(task.details.deadline).toEqual('2021-01-01');
```

Comment:
prefer that we commit to using chai everywhere (since we're mostly doing this now).

Reply:
sure